packages/salesforcedx-vscode-visualforce/package.json

```diff
@@ -53,7 +53,6 @@
     "test": "node ./node_modules/vscode/bin/test"
   },
   "activationEvents": ["workspaceContains:sfdx-project.json"],
-  "enableProposedApi": true,
```

Comment:
I'm still able to compile and when I generate the .vsix, I don't see any loss in functionality.